projects/storefrontlib/src/cms-components/checkout/components/shipping-address/shipping-address.component.html

```diff
@@ -23,7 +23,13 @@ <h3 class="cx-checkout-title d-none d-lg-block d-xl-block">
         </div>
       </div>
 
-      <div class="cx-checkout-body row">
+      <div
+        class="cx-checkout-body row"
+        *ngIf="
+          !(changeSelectedAddressLoaderState$ | async).loading;
+          else loading
+        "
+      >
```

Comment:
What is the expected behaviour?  Is is to display a spinner right after a delivery address is selected?

If so, it does not seem to work as intended.  When I select an address, it can take a while before the spinner his displayed.